src/data/set/basic.lean

```diff
@@ -492,9 +495,11 @@ ext $ assume a, by simp [or.comm, or.left_comm]
 @[simp] theorem union_insert : s ∪ insert a t = insert a (s ∪ t) :=
 ext $ assume a, by simp [or.comm, or.left_comm]
 
--- TODO(Jeremy): make this automatic
+theorem insert_nonempty (a : α) (s : set α) : (insert a s).nonempty :=
+⟨a, mem_insert a s⟩
+
 theorem insert_ne_empty (a : α) (s : set α) : insert a s ≠ ∅ :=
-by safe [ext_iff, iff_def]; have h' := a_1 a; finish
+(insert_nonempty).ne_empty
```

Comment:
missing explicit arguments

Reply:
This fix was lost while `git rebase`ing. That's one of the reasons I'd prefer if mathlib did proper `git merge` instead of squashing the history (then `git merge` would work).